testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/mdb/JMSMessageDrivenBeanTestCase.java

```diff
@@ -58,7 +59,8 @@
     public static Archive<?> deployment() {
         final Archive<?> deployment = ShrinkWrap.create(JavaArchive.class, "JMSMessageDrivenBeanTestCase.jar")
                 .addClasses(ReplyingMDB.class,JMSMessagingUtil.class)
-                .addClass(CreateQueueSetupTask.class);
+                .addClass(CreateQueueSetupTask.class)
+                .addClass(TimeoutUtil.class);
```

Comment:
Note this is the same comment as my last one. github made it disappear for a while so i retyped it. :)